policy/release/sbom_cyclonedx_test.rego

```diff
@@ -146,6 +146,8 @@ assert_allowed(purl, disallowed_packages) if {
 		"path": "/statement/predicate/components/0/purl",
 		"value": purl,
 	}])
+
+	# regal ignore:with-outside-test-context
```

Comment:
I think we should be fine to rename `assert_allowed` to `test_assert_allowed` instead

Reply:
I think maybe it would cause the test runner to try to run helper function as its own separate test, so it might not be quite that easy.